fvtest/compilertriltest/BitPermuteTest.cpp

```diff
@@ -20,7 +20,6 @@
  *******************************************************************************/
```

Comment:
typo in the commit message "no longer par of" -> "no longer part of"

Reply:
:man_facepalming: Fixing.